src/cryogen_core/compiler.clj

```diff
@@ -477,29 +477,43 @@
              :ignored-files (map #(re-pattern-from-ext (m/ext %)) (m/markups))}))))
 
 (defn compile-assets
-  "Generates all the html and copies over resources specified in the config"
+  "Generates all the html and copies over resources specified in the config.
+
+  Params:
+   - `overrides-and-hooks` - may contain overrides for `config.edn`; anything
+      here will be available to the page templates, except for these two special
+                parameters:
+     - `:extend-params-fn` - a function (`params`, `site-data`) -> `params` -
+                             use it to derive/add additional params for templates
+     - `:update-article-fn` - a function (`article`, `config`) -> `article` to update a
+                            parsed page/post. Return nil to exclude it."
   ([]
    (compile-assets {}))
-  ([overrides]
+  ([{:keys [extend-params-fn update-article-fn]
+     :or {extend-params-fn (fn [params _] params)
+          update-article-fn identity}
```

Comment:
Oh, this can't be `identity` because it's called with 2 args, but I can fix it :)